src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -98,6 +106,42 @@ public void resendStaffNotifications(
         }
     }
 
+    @PutMapping("/claim/{referenceNumber}/claimDocumentType/{claimDocumentType}/uploadDocumentToDocumentManagement")
+    @ApiOperation("Upload document to document Management")
+    public void uploadDocumentToDocumentManagement(
+        @PathVariable("referenceNumber") String referenceNumber,
+        @PathVariable("claimDocumentType") ClaimDocumentType claimDocumentType,
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) String authorisation) {
+
+        Claim claim = claimService.getClaimByReferenceAnonymous(referenceNumber)
+            .orElseThrow(() -> new NotFoundException(String.format(CLAIM_DOES_NOT_EXIST, referenceNumber)));
+        if (StringUtils.isBlank(authorisation)) {
```

Comment:
suggestion - this may not even occur if the authorisation is required

Reply:
yeah this check helps because even if we make Authorisation as required,  it still does not stop someone from sending a blank.